collectors/logsmanagement.plugin/plugin_logsmanagement_web_log.c

```diff
@@ -0,0 +1,995 @@
+                                chart_data->num_bandwidth_req_size);
+        rrddim_set_by_pointer(  chart_data->st_bandwidth, 
+                                chart_data->dim_bandwidth_resp_size, 
+                                chart_data->num_bandwidth_resp_size);
+        rrdset_done(chart_data->st_bandwidth);
+    }
+
+    /* Request proc time - update chart */
+    if(p_file_info->parser_config->chart_config & CHART_REQ_PROC_TIME){
+        if(likely(!first_update)) rrdset_next(chart_data->st_req_proc_time);
+        rrddim_set_by_pointer(  chart_data->st_req_proc_time, 
+                                chart_data->dim_req_proc_time_min, 
+                                chart_data->num_req_proc_time_min);
+        rrddim_set_by_pointer(  chart_data->st_req_proc_time, 
+                                chart_data->dim_req_proc_time_max, 
+                                chart_data->num_req_proc_time_max);
+        rrddim_set_by_pointer(  chart_data->st_req_proc_time, 
+                                chart_data->dim_req_proc_time_avg, 
+                                chart_data->num_req_proc_time_avg);
+        rrdset_done(chart_data->st_req_proc_time);
+    }
+
+    /* Response code family - update chart */
+    if(p_file_info->parser_config->chart_config & CHART_RESP_CODE_FAMILY){
+        if(likely(!first_update)) rrdset_next(chart_data->st_resp_code_family);
+        rrddim_set_by_pointer(  chart_data->st_resp_code_family, 
+                                chart_data->dim_resp_code_family_1xx, 
+                                chart_data->num_resp_code_family_1xx);
+        rrddim_set_by_pointer(  chart_data->st_resp_code_family, 
+                                chart_data->dim_resp_code_family_2xx, 
+                                chart_data->num_resp_code_family_2xx);
+        rrddim_set_by_pointer(  chart_data->st_resp_code_family, 
+                                chart_data->dim_resp_code_family_3xx, 
+                                chart_data->num_resp_code_family_3xx);
+        rrddim_set_by_pointer(  chart_data->st_resp_code_family, 
+                                chart_data->dim_resp_code_family_4xx, 
+                                chart_data->num_resp_code_family_4xx);
+        rrddim_set_by_pointer(  chart_data->st_resp_code_family, 
+                                chart_data->dim_resp_code_family_5xx, 
+                                chart_data->num_resp_code_family_5xx);
+        rrddim_set_by_pointer(  chart_data->st_resp_code_family, 
+                                chart_data->dim_resp_code_family_other, 
+                                chart_data->num_resp_code_family_other);
+        rrdset_done(chart_data->st_resp_code_family);
+    }
+
+    /* Response code - update chart */
+    if(p_file_info->parser_config->chart_config & CHART_RESP_CODE){
+        if(likely(!first_update)) rrdset_next(chart_data->st_resp_code);
+        for(int j = 0; j < 501; j++) rrddim_set_by_pointer( chart_data->st_resp_code, 
```

Comment:
Now that I am seeing this code again, probably we need to have here  a dictionary. :thinking: